easybuild/easyconfigs/j/jupyterlmod/jupyterlmod-3.0.0-GCCcore-11.3.0.eb

```diff
@@ -0,0 +1,45 @@
+easyblock = 'PythonBundle'
+
+name = 'jupyterlmod'
+version = '3.0.0'
+
+homepage = 'https://pypi.org/project/jupyterlmod/'
+description = """Jupyter interactive notebook server extension that allows user to select software modules
+ to load with Lmod before launching kernels."""
+
+toolchain = {'name': 'GCCcore', 'version': '11.3.0'}
+
+builddependencies = [
+    ('binutils', '2.38'),
+]
+
+dependencies = [
+    ('Python', '3.10.4'),
+    ('JupyterHub', '3.0.0'),
```

Comment:
@casparvl jupyterlmod does not really depend on the hub. It has optional dependencies on `jupyter-server-proxy` and `jupyterlab-server-proxy`, but IMO those are not really necessary to use it. The notebook extension works fine with IPython, and the lab extension works well with just JupyterLab. So I would leave these optional dependencies as optional, to be loaded by the user.

I made a PR for both the nothebook and lab extension of jupyter-lmod in https://github.com/easybuilders/easybuild-easyconfigs/pull/16563 . Please check them out and let me know if they work for you. We could make this PR depend on that one.

Reply:
Sorry, completely overlooked this comment, and I was on leave for the past 10 weeks... I had a look at your PR but had some trouble there with `jupyterlab-lmod` there (I'll comment there). Nonetheless, the `jupyterlmod` from there works fine. I'm fine using that as a dep, and indeed, I agree it actually doesn't make sense to include JupyterHub as dependency.